doc/Development_Documentation/20_Extending_Pimcore/13_Bundle_Developers_Guide/08_Adding_Document_Types.md

```diff
@@ -41,3 +39,9 @@ pimcore:
             map:
                 book: \App\Model\Document\Book
 ```
+
+### Do not use to override a class
+
+The type_definitions should only be used to add new documents.
+If you want to override an existing class please use the `pimcore:models:class_overrides` instead.
+For a more detailed explanation see [Overriding Models](https://github.com/pimcore/pimcore/tree/11.x/doc/Development_Documentation/20_Extending_Pimcore/03_Overriding_Models.md).
```

Comment:
```suggestion
For a more detailed explanation see [Overriding Models](../../03_Overriding_Models.md).
```
should be enough, please also verify why the 11.x doc is missing, potentially a remote tracking merge problem from 10.6 into 11

Reply:
https://github.com/pimcore/pimcore/blob/11.x/doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/08_Adding_Document_Types.md I don't see a problem with the 11.x?

Reply:
- that one is the right one, the former before change is 
https://github.com/pimcore/pimcore/tree/11.x/doc/Development_Documentation/20_Extending_Pimcore/03_Overriding_Models.md

- I noticed that i've suggested wrong, it seems there's an extra `../`